src/index.js

```diff
@@ -38,6 +39,21 @@ export default ({
 
     return new Promise((resolve, reject) => {
       const promiseCreators = action[CHAIN_API].map((createCallApiAction) => {
+        const apiAction = createCallApiAction()[CALL_API]
+        if (apiAction.revalidationEnabled) {
+          if (!apiAction.revalidate) {
+            return () => Promise.resolve()
+          }
+          const revalidationKey = _getRevalidationKey(apiAction)
+          const lastExecutedTime = lastExecutionTimeMap[revalidationKey] || 0
```

Comment:
How about `lastRevalidateTime` to unify our languages?

Reply:
Updated to https://github.com/CodementorIO/redux-api-middleman/pull/36